app/models/invoice.rb

```diff
@@ -68,7 +68,8 @@ class Invoice < ApplicationRecord
     end
   end
 
-  sequenced scope: ->(invoice) { invoice.customer.invoices }
+  sequenced scope: ->(invoice) { invoice.customer.invoices },
+            organization_scope: ->(invoice) { invoice.organization.invoices.where(created_at: Time.now.all_month) }
```

Comment:
Rails/TimeZone: Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.xmlschema`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.httpdate`, `Time.now.to_i`, `Time.now.to_f` instead. (https://rails.rubystyle.guide#time, http://danilenko.org/2012/7/6/rails_timezones)